src/test/cypress/support/constants.ts

```diff
@@ -17,8 +17,8 @@ export const TIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss.SSS';
 
 // Exercise types
 export enum EXERCISE_TYPE {
-    Text,
-    Modeling,
-    Quiz,
-    Programming,
+    Text = 'text',
```

Comment:
Why are these changes necessary?

Reply:
They are not relevant at the moment, but will be when I switch to using the exercise type enum from within the client code instead of building a separate enum:
https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/entities/exercise.model.ts#L33-L39

This will be done in a follow up. Thanks for the hin!